packages/webapp/src/components/Task/AddSoilAmendmentProducts/QuantityApplicationRate/useQuantityApplicationRate.ts

```diff
@@ -63,7 +63,7 @@ export const useQuantityApplicationRate = ({
 
   /* Update application area + rate based on percent of location */
   const onPercentLocationChange = (percent_of_location: number) => {
-    const calculatedArea = (total_area * (percent_of_location || 100)) / 100;
+    const calculatedArea = (total_area * Math.min(percent_of_location || 100, 100)) / 100;
```

Comment:
I'm not sure if this is the intended behavior as I don't have full context of this feature, but If `percent_of_location` is 0 then the || operator here would default it to 100. If the user enters 0 for the percent then is it just disregarded and assumed that 100% of location should be ammended?

Reply:
Kind of -- it's not allowed to be zero, so if the user enters zero specifically, `onBlur` it will get set to a very small minimum. I don't think we want to have the application area ever calculated as zero is why I'm not too concerned if it is temporarily at 100% before it gets bumped to the minimum.